unittests/test_f_j.py

```diff
@@ -110,8 +111,10 @@ def test_frmnam():
     assert cs.frmnam(13000) == "ITRF93"
 
 
-def test_furnsh():
-    cs.furnsh(CoreKernels.testMetaKernel)
+@checking_pathlike_filename_variants("modifier")
```

Comment:
How about something like `path_type_variant`?

Reply:
Done.  I like that.